dataframely/columns/_base.py

```diff
@@ -97,7 +102,25 @@ def validation_rules(self, expr: pl.Expr) -> dict[str, pl.Expr]:
         if not self.nullable:
             result["nullability"] = expr.is_not_null()
         if self.check is not None:
-            result["check"] = self.check(expr)
+            if isinstance(self.check, dict):
+                for k, v in self.check.items():
+                    result[k] = v(expr)
+
+            elif isinstance(self.check, list):
+                for num, i in enumerate(self.check):
```

Comment:
```suggestion
                for i, callable in enumerate(self.check):
```
I'd find this more intuitive